src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/filters/suppresswithplaintextcommentfilter/Example9.java

```diff
@@ -0,0 +1,30 @@
+/*xml
+<module name="Checker">
+  <module name="com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck">
+      <property name="max" value="100"/>
+  </module>
+  <module name="SuppressWithPlainTextCommentFilter">
+    <property name="offCommentFormat" value='=\s+"""'/>
+    <property name="onCommentFormat" value='^\s+""";'/>
+  </module>
+</module>
+
+
+*/
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.filters.suppresswithplaintextcommentfilter;
+
+// xdoc section -- start
+public class Example9 {
+
+  // ok, opening and closing triple quotes are our delimiters
```

Comment:
`// ok, opening and closing triple quotes are our on-off markers`

Reply:
Done, I used the actual property names